poc/vdaf_poc/daf.py

```diff
@@ -51,6 +104,7 @@ class Daf(
     # Number of random bytes consumed by `shard()`.
     RAND_SIZE: int
 
+    @override
```

Comment:
What's the purpose of this decorator?

Reply:
Static checkers will warn you if you have a method decorated with `@override` that is not actually overriding a superclass method. I added this as a bit of extra insurance against typos and inconsistent changes.